src/main/java/com/techiteasy/demo/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,65 @@
+package com.techiteasy.demo.controllers;
+
+import com.techiteasy.demo.exceptions.RecordNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.*;
+
+//Stap 2: voorzie de klasse van de juiste annotatie
+@RestController
+@RequestMapping
```

Comment:
RequestMapping heeft opsich geen toegevoegde waarde. Je kunt er tussen haakjes achter invullen wat de base waarde van je request paden moet zijn. Als je dus `@RequestMapping("/televisions")` invult, dan komt jou "getAllTelevisions" op het pad "localhost:8080/televisions" in plaats van slechts "localhost:8080"